core/src/main/java/tc/oc/pgm/command/MapCommand.java

```diff
@@ -115,6 +119,20 @@ public Component format(MapInfo map, int index) {
     }.display(audience, ImmutableList.copyOf(maps), page);
   }
 
+  @Suggestions("maptags")
+  public List<String> suggestMapTags(CommandContext<CommandSender> sender, String input) {
+    int commaIdx = input.lastIndexOf(',');
+
+    final String prefix = input.substring(0, commaIdx == -1 ? 0 : commaIdx + 1);
+    final String toComplete =
+        input.substring(commaIdx + 1).toLowerCase(Locale.ROOT).replace("!", "");
+
+    return Stream.of("wool", "monument")
```

Comment:
This will only suggest wool and monument

Reply:
leftover from testing, nice catch